File: Python/Fundamentals/Functions/Exercise_Functions/perfect_number.py
def perfect_number(number):
    sum = 0
    for divisor in range(1, number):
        if number % divisor == 0:
            sum += divisor
    if number == sum:
        return "We have a perfect number!"
    return "It's not so perfect."

number = int(input())
print(perfect_number(number))

File: Python/JS-Front-End/Functions-and-statements/Exercise/perfect-number.js
function solve(number) {

    let perfectNum = 0;

    for (let index = 1; index < number; index++) {

        if (number % index === 0) {
            perfectNum += index
        }
        
    }

    if (number === perfectNum) {
        console.log("We have a perfect number!")
    } else {
        console.log("It's not so perfect.");
    }
}

solve(123);